src/sockets/handlers/transcribedTextHandler.js

```diff
@@ -1,20 +1,58 @@
 import { getAdKeywords } from "../../cache/adKeywordCache.js";
+import { getUserByIdService } from "../../services/users/userService.js";
 
-const transcribedTextHandler = (whisperSocket, io, userMap) => {
-  whisperSocket.off("transcribedRadioText");
+let isAdPlaying = false;
+let adEndTimers = new Map();
 
-  whisperSocket.on("transcribedRadioText", ({ text, userId }) => {
+const handleTranscribedText =
+  (io, userMap) =>
+  async ({ text, userId }) => {
     console.log("[Whisper] Received text:", text);
     const socketId = userMap.get(userId);
+    if (!socketId) return;
```

Comment:
```suggestion
    if (!socketId) {
      return;
    }
```
중괄호 넣어주면 더 좋을 것 같아요!

Reply:
감사합니다!
중괄호 반영했습니다. ☺️